test/dependency_validator/v1_updater_test.dart

```diff
@@ -0,0 +1,317 @@
+                expectedOutput: contextWrapper(
+                    '$depValidatorCommand --ignore over_react,$addedDependency'),
+              );
+            });
+
+            test('using `--ignore=`', () async {
+              await testSuggestor(
+                shouldDartfmtOutput: false,
+                expectedPatchCount: 1,
+                input:
+                    contextWrapper('$depValidatorCommand --ignore=over_react'),
+                expectedOutput: contextWrapper(
+                    '$depValidatorCommand --ignore=over_react,$addedDependency'),
+              );
+            });
+
+            test('when there happens to be a trailing comma', () async {
+              await testSuggestor(
+                shouldDartfmtOutput: false,
+                expectedPatchCount: 1,
+                input: contextWrapper('$depValidatorCommand -i over_react,'),
+                expectedOutput: contextWrapper(
+                    '$depValidatorCommand -i over_react,$addedDependency'),
+              );
+            });
+
+            test('when there are args after the ignore arg', () async {
+              await testSuggestor(
+                shouldDartfmtOutput: false,
+                expectedPatchCount: 1,
+                input: contextWrapper(
+                    '$depValidatorCommand -i over_react -x app/,bin/,random/ --no-fatal-missing'),
+                expectedOutput: contextWrapper(
+                    '$depValidatorCommand -i over_react,$addedDependency -x app/,bin/,random/ --no-fatal-missing'),
+              );
+            });
+
+            test('when there are args before the ignore arg', () async {
+              await testSuggestor(
+                shouldDartfmtOutput: false,
+                expectedPatchCount: 1,
+                input: contextWrapper(
+                    '$depValidatorCommand -x app/,bin/,random/ --no-fatal-missing -i over_react'),
+                expectedOutput: contextWrapper(
+                    '$depValidatorCommand -x app/,bin/,random/ --no-fatal-missing -i over_react,$addedDependency'),
+              );
+            });
+
+            test('when there are args on both sides of the ignore arg',
+                () async {
+              await testSuggestor(
+                shouldDartfmtOutput: false,
+                expectedPatchCount: 1,
+                input: contextWrapper(
+                    '$depValidatorCommand -x app/,bin/,random/ -i over_react --no-fatal-missing'),
+                expectedOutput: contextWrapper(
+                    '$depValidatorCommand -x app/,bin/,random/ -i over_react,$addedDependency --no-fatal-missing'),
+              );
+            });
+          });
+        });
+      });
+    }
+
+    group('within the context', () {
+      for (final context in contexts.keys) {
+        sharedCommandTests(context);
+      }
+    });
+
+    test('adds to all locations within a file', () async {
+      final context = contexts['basic-dockerfile'];
```

Comment:
#nit inconsistent naming with other places, where `context` is the key in the map and not the value.